code/game/machinery/scanner_gate.dm

```diff
@@ -251,12 +276,15 @@
 
 	use_energy(active_power_usage)
 
-/obj/machinery/scanner_gate/proc/alarm_beep()
-	if(next_beep <= world.time)
-		next_beep = world.time + (2 SECONDS)
-		playsound(src, 'sound/machines/scanbuzz.ogg', 100, FALSE)
-	var/mutable_appearance/alarm_display = mutable_appearance(icon, "alarm_light")
-	flick_overlay_view(alarm_display, 2 SECONDS)
+/obj/machinery/scanner_gate/proc/alarm_beep(detected_thing)
+	if(!COOLDOWN_FINISHED(src, next_beep))
+		return
+
+	if(detected_thing)
+		src.say("[reverse ? "Not" : "A"] [detected_thing] person detected!!")
```

Comment:
instead of saying what "kind" of person they are: say what the scanner detected, Example: "Romerol infection detected" or "Weapons detected"

Reply:
The scanner does only scan carbons so why though? Its always a person

Reply:
if it's always a person then why clarify it?